operations/c8y_MapUpdateRegister.template

```diff
@@ -0,0 +1,7 @@
+[exec]
+  topic = "c8y/devicecontrol/notifications"
+  on_fragment = "c8y_SetRegister"
+
+[exec.workflow]
+operation = "c8y_SetRegister"
+input.c8y_SetRegister = "${.payload}"
```

Comment:
Is this working? If yes, then I'm missing something.

Looking the code of `parse_register_params`, the plugin expect a JSON object `{"input", "ipAddress", "address", "register", "startBit", "noBits", "value" }`. Not a fragment names `c8y_SetRegister`.

Hence, I would rewrite this as:

```suggestion
input = "${.payload}.c8y_SetRegister"
```

i.e. to pass only the  `c8y_SetRegister` content to the operation.